src/config.ts

```diff
@@ -140,7 +140,15 @@ const DEFAULTS = o({
     }),
     autocmds: o({
         DocStart: o({
-            // "addons.mozilla.org": "mode ignore",
+            /** chocolateboy's pagination fixes */
```

Comment:
@chocolateboy points out that we should add a link back to https://github.com/chocolateboy/userscripts#pagerizers